### ISSUE TITLE
Creating Gif via replay episode returns an error 

### ISSUE THREAD

**Issue by ML-IEE** [rte-france/Grid2Op#569]
## Environment
 - Grid2op version: `1.9.5`

## Bug description
Hey, I am not quite sure whether or not this might already been fixed in the newer versions, however in Grid2Op v1.9.5 I was not able to create a gif due to the depriciated argument "fps". I could not see in the Release note, whether this was fixed. 

## How to reproduce
The code is quite similar to the [plotting example](https://grid2op.readthedocs.io/en/latest/plot.html)
In the following example we import the results from an Runner 
```python
import grid2op
from grid2op.Episode import EpisodeReplay
env = grid2op.make("l2rpn_wcci_2022")

plot_epi = EpisodeReplay("path")
plot_epi.replay_episode("2050-08-08_5",gif_name = "08-08-05",
                        start_step = 385, end_step = 700)
```

## Possible Error 
As far as I can see the error is due to the depreciated fps argument in the .replay_episode() method
```python
imageio.mimwrite(gif_path, frames, fps=fps)
```
## Proposed fix
The fix is rather simple. Just use duration instead of fps 
```python
imageio.mimwrite(gif_path, frames, duration=1000/fps)
```

**Comment by BDonnot** [comment]
Hi, 

Thanks for noticing it. I added a `try / catch` block to make sure the old behaviour is still correct on most ancient imageio version, but beside that your fix seems good :-) 

Thanks !

**Comment by BDonnot** [comment]
Should be solved in 1.9.8
